lib/matplotlib/backends/backend_pgf.py

```diff
@@ -790,8 +790,9 @@ class FigureCanvasPgf(FigureCanvasBase):
     def get_default_filetype(self):
         return 'pdf'
 
-    def _print_pgf_to_fh(self, fh, *args, **kwargs):
-        if kwargs.get("dryrun", False):
+    def _print_pgf_to_fh(self, fh, *args,
```

Comment:
*args is unused -> * only

Reply:
see above re: keeping the same signature for all print_foo's.  In any case this would mean fixing the call sites as well, so it'll be separate work.

Reply:
ok.